client/utils.ts

```diff
@@ -0,0 +1,27 @@
+// Adapts route paths to also include wildcard
+// subroutes under the root level section.
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function pathToRegExp( path: any ) {
```

Comment:
Instead of using `any` here, we should be able to determine what type this is. For example:

```ts
export function pathToRegExp( path: string ) {
```